packages/node/src/index.ts

```diff
@@ -506,6 +506,9 @@ export const build: BuildV3 = async ({
       shouldAddSourcemapSupport,
       awsLambdaHandler,
       supportsResponseStreaming,
+      ...(staticConfig?.maxDuration
+        ? { maxDuration: staticConfig.maxDuration }
+        : {}),
```

Comment:
Is it necessary to spread? Can you do this instead:

```suggestion
      maxDuration: staticConfig?.maxDuration
```

Reply:
Yeah we can also do that 👍🏻